client/src/components/playerList.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import axios from "axios"
+import PlayerCard from "./playerCard"
+
+
+
+
+class PlayerList extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            players:[]
+        }
+    }
+
+    componentDidMount() {
+    axios.get(`http://localhost:5000/api/players`)
+    .then (res => {
+      console.log(res.data)
+      var playersData = res.data
+      this.setState({
+        players:playersData
+            })            
+        })
+    .catch(err => console.log('ERROR'));
+    }
+
+    
+
+    render(props) {
+        return (
+                <div data-testid ='card-container' className='card-container'>
+                   {this.state.players.map(player => (
+                       <div className='player'>
+                        <PlayerCard key={player.id} name={player.name} country={player.country} searches={player.searches} />
+                    
+                       </div>
```

Comment:
Check your terminal here to see if it;s still giving you a warning - I think the `key={player.id}` needs to be on the parent component inside the `map()`, which means it would need to be on the `<div>` like so:
```
<div key={player.id} className='player'>
    <PlayerCard name={player.name} country={player.country} searches={player.searches} />
</div>
```